elink, esearch: add page (#8957)
tldr-pages/tldr: pages/linux/elink.md

# elink

> > Elink looks up precomputed neighbors within a database, or finds associated records in other databases.

Reviewer: ```suggestion
> Look up precomputed neighbors within a database, or find associated records in other databases.
```

> > It is part of the edirect package: <https://www.ncbi.nlm.nih.gov/books/NBK179288/>.

Reviewer 2: ```suggestion
> It is part of the edirect package.
> More information: <https://www.ncbi.nlm.nih.gov/books/NBK179288/>.
```

> - Search pubmed and then find related sequences:

Reviewer: ```suggestion
- Search pubmed then find related sequences:
```

> `esearch -db pubmed -query "selective serotonin reuptake inhibitor" | elink -target nuccore`

Reviewer: ```suggestion
`esearch -db pubmed -query "{{selective serotonin reuptake inhibitor}}" | elink -target nuccore`
```

> - Search nucleotide and then find related biosamples:

Reviewer: ```suggestion
- Search nucleotide then find related biosamples:
```

> `esearch -db nuccore -query "insulin [PROT] AND rodents [ORGN]" | elink -target biosample`

Reviewer: ```suggestion
`esearch -db nuccore -query "{{insulin [PROT] AND rodents [ORGN]}}" | elink -target biosample`
```

# elink

> Look up precomputed neighbors within a database, or find associated records in other databases.

> > It is part of the edirect package.

Reviewer: ```suggestion
> It is part of the `edirect` package.
```
